src/processing/ClientBoardState.java

```diff
@@ -14,61 +17,232 @@
 
 public class ClientBoardState {
 	
-	//maps is an object of the BoardState containing both the maps
+	/**
+	 * maps is an object of BoardState which stores both the maps which
+	 * we need for persistence on the server.
+	 */
 	public static BoardState maps = new BoardState();
 	
-	//port Number of the Board Server
+	/**
+	 * portNumber stores the port number of the Board Server to make 
+	 * communications from the server
+	 */
 	public static Port portNumber;
 	
-	//public static Map <Identifier identifier, IChanges handler>
-	
+	/** boardId will store the Board ID of the board to which
+	 * the client is connected to.
+	 */
 	public static BoardId boardId;
```

Comment:
I think you have to make this synchronized

Reply:
Why only once it will be set